packages/lodestar-fork-choice/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@chainsafe/lodestar-fork-choice",
+  "description": "A Typescript implementation eth2 fork choice",
+  "license": "Apache-2.0",
+  "author": "ChainSafe Systems",
+  "homepage": "https://github.com/ChainSafe/lodestar#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com:ChainSafe/lodestar.git"
+  },
+  "bugs": {
+    "url": "https://github.com/ChainSafe/lodestar/issues"
+  },
+  "version": "0.11.0",
+  "main": "lib/index.js",
+  "files": [
+    "lib/**/*.d.ts",
+    "lib/**/*.js",
+    "lib/**/*.js.map"
+  ],
+  "scripts": {
+    "benchmark": "ts-node test/benchmarks",
+    "build": "yarn run build:lib && yarn run build:types",
+    "build:typedocs": "typedoc --exclude src/index.ts --out typedocs src",
+    "build:lib": "babel src -x .ts -d lib --source-maps",
+    "build:lib:watch": "yarn run build:lib --watch",
+    "build:release": "yarn clean && yarn run build && yarn run build:typedocs",
+    "build:types": "tsc --incremental --declaration --outDir lib --project tsconfig.build.json --emitDeclarationOnly",
+    "build:types:watch": "yarn run build:types --watch --preserveWatchOutput",
+    "check-types": "tsc --noEmit",
+    "clean": "rm -rf lib && rm -f tsconfig.tsbuildinfo && rm -f tsconfig.build.tsbuildinfo",
+    "coverage": "codecov -F lodestar-fork-choice",
+    "lint": "eslint --color --ext .ts src/",
+    "lint:fix": "eslint --color --ext .ts src/ --fix",
+    "pretest": "yarn run check-types",
+    "test": "yarn test:unit && yarn test:e2e",
+    "test:e2e": "mocha --colors -r ts-node/register 'test/e2e/**/*.test.ts'",
+    "test:unit": "mocha --colors -r ts-node/register 'test/unit/**/*.test.ts'"
+  },
+  "dependencies": {
+    "@chainsafe/lodestar-beacon-state-transition": "^0.11.0",
+    "@chainsafe/lodestar-config": "^0.11.0",
+    "@chainsafe/lodestar-params": "^0.11.0",
+    "@chainsafe/lodestar-types": "^0.11.0",
+    "@chainsafe/lodestar-utils": "^0.11.0",
+    "@chainsafe/ssz": "^0.6.13"
+  },
+  "devDependencies": {
+    "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
+    "mocha": "^6.2.2"
```

Comment:
do we need those since they are already in root package.json.
If we keep them all in one place, it's easier to keep same version in every package